mono/utils/monobitset.h

```diff
@@ -9,26 +9,15 @@
 #include <mono/utils/mono-publib.h>
 #endif
 
-/*
```

Comment:
There was somewhere that included one or the other based on an `#ifdef`. I will check it out.